### ISSUE TITLE
Jobs always get killed/aborted at 7:34pm

### ISSUE THREAD

**Issue by sebw** [rundeck/rundeck#2128]
# Bug report

**My Rundeck detail**
- Rundeck version: 2.6.8
- install type: RPM from repo
- OS Name/version: RHEL 6.7 x86_64

**Expected Behavior**

Running jobs should not get killed at 7:34pm

**Actual Behavior**

Jobs get killed at 7:34pm manually executed at different times of the day. Job is reported as aborted in the logs, while killed in the web GUI.

This is the result of "grep f9b79957-7ac3-4fb7-804a-b2661d2f1a0c rundeck.exec*":

```
rundeck.executions.log.2016-08-08:[2016-08-08 11:29:16,200] sebw start [1148163:running] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-08-08:[2016-08-08 19:34:31,699] sebw finish [1148163:aborted] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-08-23:[2016-08-23 11:31:22,442] sebw start [1177662:running] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-08-23:[2016-08-23 19:34:36,600] sebw finish [1177662:aborted] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-08-24:[2016-08-24 08:59:34,676] sebw start [1179426:running] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-08-24:[2016-08-24 19:34:34,130] sebw finish [1179426:aborted] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-10-19:[2016-10-19 10:52:17,568] sebw start [1288218:running] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-10-19:[2016-10-19 19:34:33,952] sebw finish [1288218:aborted] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-10-24:[2016-10-24 09:59:44,196] sebw start [1297755:running] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-10-24:[2016-10-24 10:01:49,125] sebw finish [1297755:succeeded] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-10-24:[2016-10-24 11:09:51,533] sebw start [1297851:running] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-10-24:[2016-10-24 11:11:33,912] sebw finish [1297851:succeeded] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-10-24:[2016-10-24 11:14:28,424] sebw start [1297857:running] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
rundeck.executions.log.2016-10-24:[2016-10-24 19:34:50,704] sebw finish [1297857:aborted] PROJECT sebw/- "Solr full reindex"[f9b79957-7ac3-4fb7-804a-b2661d2f1a0c]
```

I'm getting the same result with a long running command executed from the GUI (basically an infinite sleep):

```
rundeck.executions.log.2016-10-24:[2016-10-24 13:38:30,693] sebw start [1298040:running] COMPANY sebw/- "-/-"[-]
rundeck.executions.log.2016-10-24:[2016-10-24 19:34:57,274] sebw finish [1298040:aborted] COMPANY sebw/- "-/-"[-]
```

After investigating a bit more, I noticed that some scheduled jobs are impacted as well.

```
# grep '\ 19:34' rundeck.exec* | grep aborted | wc -l
47
```

No other jobs are scheduled to run at 7:34pm nor any local crons. I was thinking maybe about log rotation but rotation occurs at midnight.

**How to reproduce Behavior**

Run a job that is supposed to be running past 7:34pm.

I'm trying to reproduce the bug on a fresh RHEL7 install, I'll keep you posted.

**Comment by sebw** [comment]
I can't reproduce this problem on RHEL7 but it is running 2.6.9.
I will update to 2.6.9 on RHEL6 and see if it helps.

**Comment by sebw** [comment]
The problem was still present with 2.6.11. We decided to migrate to RHEL7 as this was becoming blocking for long running jobs.

I confirm everything is fine now.

After the migration, I kept the old RHEL6 server powered on but with the service stopped and disabled at boot.

The next morning I noticed Rundeck was running since 7:33pm. Obviously no one has been working that evening. Fortunately I had some iptables rules blocking any trafic Rundeck could generate.

I'm really missing something here, but it is obviously a problem on my side and a problem on that server more than with RHEL6.

Closing.
